### ISSUE TITLE
How to deal with foreign key constraint?

### ISSUE THREAD

**Issue by osdakira** [sue445/heart_seed#27]
```rb
$ bin/rake db:seed

ActiveRecord::StatementInvalid: Mysql2::Error: Cannot delete or update a parent row: a foreign key constraint fails 
```

Does the heart_seed have `UPDATE` sql instead of `delete all` and `bulk insert` ?

Or ignore `foreign_key_checks` option ?

For example

```rb
ActiveRecord::Base.connection.execute "SET FOREIGN_KEY_CHECKS = 0"
heart_seed:db:seed
ActiveRecord::Base.connection.execute "SET FOREIGN_KEY_CHECKS = 1"
```

**Comment by sue445** [comment]
Oh. I did not consider foreign key.
I'll try consider a good way.

**Comment by sue445** [comment]
@osdakira I released `v0.1.0` !!
